### ISSUE TITLE
[WebToolsE2E][Aspire] Click Help on the Aspire Dashboard page, and the "Close" button prompt will automatically appear.

### ISSUE THREAD

**Issue by v-reinawang** [dotnet/aspire#6021]
**REGRESSION INFO**: Worked in Aspire 8.2.1

**INSTALL STEPS**
1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.402
3. Install Aspire 9.0 RC1 [9.0.0-preview.4.24480.1]

**REPRO STEPS**

1. Open CMD, run following command to create an Aspire Starter App project
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`

**ACTUAL**
Click Help, the prompt ‘Close’ will be automatically displayed.
![dashboard3](https://github.com/user-attachments/assets/7b9f71d8-1a97-46e5-9677-f1a166cac589)

Also reproduce the Add filter page in Structured and Traces
![image](https://github.com/user-attachments/assets/2cd15e42-c7c6-4c44-9e1b-8a740f238fb5)

**EXPECTED**
When the mouse hovers over the close button, the prompt "Close" will appear.

**Comment by adamint** [comment]
This is a regression in fluentui-blazor, this appears in examples [here](https://www.fluentui-blazor.net/Dialog)

@v-reinawang could you file a ticket in microsoft/fluentui-blazor

**Comment by v-reinawang** [comment]
I filed an [issue](https://github.com/microsoft/fluentui-blazor/issues/2787) in https://github.com/microsoft/fluentui-blazor

**Comment by v-reinawang** [comment]
This issue has been fixed in latest Aspire 9.1.0-preview.1.24578.4.
